Stresser/RegistryEventHandler.cpp

```diff
@@ -6,12 +6,18 @@
 #include "../Utils/RegistryArtifactUtils.h"
 #include "EventEntity.h"
 
-RegistryEventHandler::RegistryEventHandler() : IEtwEventHandler(EtwEventTypes::Registry)
+RegistryEventHandler::RegistryEventHandler(std::vector<IArtifact*>& artifacts) :
+IEtwEventHandler(EtwEventTypes::Registry), m_artifacts(artifacts)
 {
 }
 
 void RegistryEventHandler::onEventRecord(PEVENT_RECORD record)
 {
+	if(m_artifacts.empty())
+	{
+		return;
+	}
+
```

Comment:
Like in `FileEventHandler`, please verify that this `record` contains the specific corresponding event type for this specific handler, by the `opcode` filed.

Make the `opcode` a constant value please 😄
Maybe put it in the header